src/doc/style-guide/src/expressions.md

```diff
@@ -328,6 +328,37 @@ foo_bar
 Prefer line-breaking at an assignment operator (either `=` or `+=`, etc.) rather
 than at other binary operators.
 
+If line-breaking a binary operator (including assignment operators) where the
+first operand spans multiple lines, use the base indentation of the *last*
+line of the first , and indent relative to that:
```

Comment:
Recognize it's unnecessary and a bit repetitive, but given the usage of last and first in short succession I feel like it'd be helpful to include this: 
```suggestion
line of the first operand, and indent relative to that:
```

if anyone strongly objects though, then we should at least drop the extra space:
```suggestion
line of the first, and indent relative to that:
```

Reply:
Yeah, this was a typo. Thank you. ❤️